### ISSUE TITLE
Perform Some Analysis on the Results

### ISSUE THREAD

**Issue by KyleS22** [KyleS22/Unveiling-the-potential-of-GNN-for-network-modeling-and-optimization-in-SDN#5]
At this point, I should have a metric tonne of results from the various trainings of the model.  I'll need to create visualizations:
   - I'd like to have a comparison of accuracy between which of the data sets was used to train
   - Some sort of comparison of the hyper-parameters and accuracy for each of the training data sets

I might want to perform some form of statistical analysis at this point, we will see what the data looks like.
